### ISSUE TITLE
functions are not appearing in namespace

### ISSUE THREAD

**Issue by sanjmeh** [lorenzwalthert/gitsum#44]
I find several functions that are in the documentation not available in namespace even when explicitly declaring the package name on the command line.
```
gitsum::read_last_commit(path = ".")
Error: 'read_last_commit' is not an exported object from 'namespace:gitsum'
```
or

```
> gitsum::get_raw_log(path = ".") -> x1
Error: 'get_raw_log' is not an exported object from 'namespace:gitsum'
```

What could be wrong here?

**Comment by lorenzwalthert** [comment]
I think they are rightly not exported. The documentation is purely internal, so I should add a `#' @keywords internal` tag to the roxygen comment so the documnentation is not visible in the index.

**Comment by lorenzwalthert** [comment]
Fixed with 47bd5511.
